<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionQuery</class>
 <widget class="QDialog" name="VersionQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
font-size:24px;
background-color: rgb(27, 150, 208);
}
QLabel{
color:white;
}
QPushButton{
border-radius:10;
background-color: rgb(80, 80, 80);
font-size:22px;
color:rgb(255, 255, 0);
}
QPushButton::pressed{
background-color: rgb(120, 120, 120);
}</string>
  </property>
  <widget class="QPushButton" name="btn_query">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>380</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>返
回</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox QAbstractItemView::item { 
background-color:white;
min-height: 40px;
min-width: 60px;
}
QComboBox {
background-color:white;
combobox-popup: 0;
min-height: 40px; 
min-width: 60px;
}
QListView::item:selected{ 
color:white;
background-color:#4d93fd;
} 
</string>
   </property>
   <item>
    <property name="text">
     <string>人机程序版本</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>选针器驱动板</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>气阀驱动板</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电机驱动板</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>主控板</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>721</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>1</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_tips">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>版本查询中，请稍后...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
